src/tree/revisionManagement/RevisionItem.ts

```diff
@@ -17,9 +17,14 @@ export interface RevisionsItemModel extends ContainerAppsItem {
     revision: Revision;
 }
 
+const revisionStateActiveContextValue: string = 'revisionState:active';
+const revisionStateInactiveContextValue: string = 'revisionState:inactive';
+const revisionModeSingleContextValue: string = 'revisionMode:single';
+const revisionModeMultipleContextValue: string = 'revisionMode:multiple';
```

Comment:
Put at the top here like this for readability/discoverability, but totally happy to re-write this inline again if preferred

Reply:
![image](https://github.com/microsoft/vscode-azurecontainerapps/assets/36966225/8f8fec23-e0ba-43b7-8a58-c00e712a2229)